integration-tests/plugins/__tests__/inventory/products/get-product.js

```diff
@@ -0,0 +1,109 @@
+const path = require("path")
+
+const { bootstrapApp } = require("../../../../helpers/bootstrap-app")
+const { initDb, useDb } = require("../../../../helpers/use-db")
+const { setPort, useApi } = require("../../../../helpers/use-api")
+
+const {
+  ProductVariantInventoryService,
+  ProductVariantService,
+} = require("@medusajs/medusa")
+
+const adminSeeder = require("../../../helpers/admin-seeder")
+
+jest.setTimeout(30000)
+
+const { simpleProductFactory } = require("../../../factories")
+const { simpleSalesChannelFactory } = require("../../../../api/factories")
+
+const adminHeaders = { headers: { Authorization: "Bearer test_token" } }
+
+describe("Create Variant", () => {
```

Comment:
todo: We should rename the describe as it should be readable as a book, here `create variants list-products` does not seems to make sense